app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java

```diff
@@ -41,17 +44,21 @@
     private Button btn2;
     private TextView exchange;
     private DocumentReference bookDocument;
+    private boolean goodScan;
```

Comment:
exchange, goodScan, resouces are not used. Can u remove them pls?

Reply:
I get this too... goodScan I can delete but exchange and resources are both used (line 65 and 78)

Reply:
ok thanks!